apps/sherbet_service/lib/sherbet.service/contact/communication/method/email.ex

```diff
@@ -217,8 +218,8 @@ defmodule Sherbet.Service.Contact.Communication.Method.Email do
         end
     end
 
-    defp generate_key() do
-        :crypto.strong_rand_bytes(32)
+    defp generate_key(data \\ @key) do
+        :crypto.strong_rand_bytes(data)
```

Comment:
I'd suggest renaming `data` to a more intuitive name, such as `length`. Aside from that, this looks fine.